pkg/cluster/internal/create/actions/createworker/aws.go

```diff
@@ -0,0 +1,110 @@
+/*
```

Comment:
This should not be in this PR, right?

Reply:
It's part of the builder refactor to make the code multicloud, it doesn't add aws vms functionality, only adapt the existing code for eks to he builder pattern.

Reply:
got it!